src/server/index.js

```diff
@@ -33,23 +53,44 @@ export function Server({ games, db, transport }) {
   }
   transport.init(app, games);
 
-  const api = createApiServer({ db, games });
-
   return {
     app,
-    api,
     db,
 
-    run: async (port, callback) => {
+    run: async (portOrConfig, callback) => {
+      const serverRunConfig = createServerRunConfig(portOrConfig, callback);
+
+      // DB
       await db.connect();
-      let apiServer = await api.listen(port + 1);
-      let appServer = await app.listen(port, callback);
-      logger.info('listening...');
+
+      // API
+      let apiServer;
+      if (!serverRunConfig.apiPort) {
+        addApiToServer({ app, db, games });
+      } else {
+        // Run API in a separate Koa app.
+        const api = createApiServer({ db, games });
+        apiServer = await api.listen(
+          Number(serverRunConfig.apiPort),
```

Comment:
@nicolodavis Quick note here. There is no need to have `Number()` wrapper here. koa is running on 51234 port if koa.listen("51234") is called. That made sense to wrap with `Number()` when code `port + 1` used. Not now.

Reply:
Ah, I see. Ok will remove that then.